src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2339,7 +2330,8 @@ public void ShowSaveDialogAndSaveResult(object parameter)
                     _fileDialog.InitialDirectory = fi.DirectoryName;
                     _fileDialog.FileName = fi.Name;
                 }
-                else if (vm.Model.CurrentWorkspace is CustomNodeWorkspaceModel)
+
```

Comment:
Removed the "else" check here as this condition for CustomNodeWorkspaceModel should be met irrespective of the FileName being null.